app/uk/gov/hmrc/tradergoodsprofilesdatastore/models/response/GoodsItemRecords.scala

```diff
@@ -58,7 +57,6 @@ object GoodsItemRecords {
         "recordId"                 -> goodsItemRecords.recordId,
         "traderRef"                -> goodsItemRecords.traderRef,
         "comcode"                  -> goodsItemRecords.comcode,
```

Comment:
the accreditationStatus has been renamed to adviceStatus now in API - we shouldn't be removing this I suppose.

Reply:
Also, the test cases need to updated for the urls. I updated them in my PR https://github.com/hmrc/trader-goods-profiles-data-store/pull/10